shopify_python/git_utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import typing  # pylint: disable=W0611
+from git import repo
+from git.refs import head  # pylint: disable=W0611
```

Comment:
We should get in the habit of disabling by name to assist reviewers (https://github.com/Shopify/shopify_python/issues/32)